cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
```

Comment:
> making the returned channel - a "cancellation" channel - i.e. if we write to the channel the printing is cancelled (if it has not already happened) [rather than the current pattern of reading from it]

@donatello we print "Lock acquired" only if we had printed "Trying to acquire lock" hence we read from the channel to see the go routine has printed. I think cancellation channel can't fulfill this requirement.